node/overseer/overseer-gen/proc-macro/build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
```

Comment:
Is this a left over file?

Reply:
It's required, otherwise `OUT_DIR` would not be populated, which we need for some intermediate file generation.